meetings/views.py

```diff
@@ -209,6 +213,8 @@ def listattendance(request, page=1):
 
     context['lists'] = [("Past Meetings", past_mtgs),
                         ("Future Meetings", future_mtgs)]
+    if len(inprogress_mtgs) > 0:
+        context['lists'].insert(0, ("In Progress Meetings", inprogress_mtgs))
```

Comment:
```suggestion
        context['lists'].insert(0, ("Meetings In Progress", inprogress_mtgs))
```
I think it would read better to say `Meetings In Progress`. Furthermore, your changes to `site_tmpl/meeting_list.html` will do nothing if this line doesn't match it.